core/templates/components/card/card.twig

```diff
@@ -0,0 +1,72 @@
+{#
+/**
+ * @file
+ * Card Component.
+ *
+ * Basic building block component with two variants
+ *   - Horizontal: An image beside text content
+ *   - Vertical (default): An image above text content
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * -- su-card--horizontal
+ * - card_image: A full HTML entity for an image.
+ * - card_super_headline: Text heading
+ * - card_headline: Text heading
+ * - card_body: An open variable for anything. Generally this would be for
+ *     WYSIWYG editor contents.
+ * - card_cta_link: The href value for the cta field link
+ * - card_cta_attributes: Any html attributes on the cta link that need to be
+ *     added. For example: _target.
+ * - card_cta_label: The clickable text for the CTA link
+ * - card_button_link: The href property of the button component.
+ * - card_button_label: The text label of the button component.
+ */
+#}
+<div{{ attributes }} class="su-card {{ modifier_class }}">
+
+  {# Card Image #}
+  {% if card_image is not empty %}
+    {{ card_image }}
+  {% endif %}
+
+  <div class="su-card__contents">
+
+  {# CTA Super Headline #}
+  {% block block_card_super_headline %}
+    {% if card_super_headline is not empty %}
+      <h5>{{ card_super_headline }}</h5>
+    {% endif %}
+  {% endblock %}
+
+  {# CTA Headline #}
+  {% block block_card_headline  %}
+    {% if card_headline is not empty %}
+      <h3>{{ card_headline }}</h3>
+    {% endif %}
+  {% endblock %}
+
+  {# An open field for body content #}
+  {{ card_body }}
+
+  {# CTA Super Headline #}
+  {% if card_cta_link is not empty %}
+    <a {{ card_cta_attributes }} href="{{ card_cta_link }}" class="su-card__cta-link">{{ card_cta_label }}</a>
+  {% endif %}
```

Comment:
Please replace `card_cta_link` with include of (new) `action` variant of a link. Also the `{# CTA Super Headline #}` comment is probably not what you want. ;)

Reply:
This has been changed to use a link component now.